WordPress/src/main/java/org/wordpress/android/ui/mysite/tabs/MySiteTabFragment.kt

```diff
@@ -176,7 +176,9 @@ class MySiteTabFragment : Fragment(R.layout.my_site_tab_fragment),
         adapter.registerAdapterDataObserver(object : AdapterDataObserver() {
             override fun onItemRangeInserted(positionStart: Int, itemCount: Int) {
                 super.onItemRangeInserted(positionStart, itemCount)
-                recyclerView.smoothScrollToPosition(0)
+                if (itemCount == ONE_ITEM && positionStart == FIRST_ITEM) {
```

Comment:
@gitstart I wonder whether the check `itemCount == ONE_ITEM` is needed or not. There might be some cases where we might have two items added to the top, the Quick start card(Next Steps Card) and the stats button when switching to a new site. So in that case, this functionality might not work as expected, in that case too we want the recycler view to be scrolled to the top. So I think the `positionStart == FIRST_ITEM` should be enough. Wdyt?. 🤔

Reply:
Thank you for reviewing @AjeshRPai,
Will implement the solution you suggest and get back on this asap. 😊

Reply:
@gitstart I am sorry about this, I tried to reproduce the scenario and I found that this scenario doesn't exist. When we switch from a site having paid features(business plan) to a free plan or when you switch from a site in which you are author to a site in which you are admin, **activity** log and **backup** are the options which gets added and removed from the list. The **stats** list item is always shown. So the scenario I mentioned above is invalid. Sorry for the confusion.

>If we only leave positionStart == FIRST_ITEM the recycler view will always scroll to the top because when we navigate back to My Site (for example from the notification) +16 elements are added to the recycler view from position 0. The itemCount == ONE_ITEM check was put there to prevent this.

Got it. Thanks for clarifying it. I get it now. I think we are good. I am going ahead and triggering CI. Once the CI is successful, I will merge the PR 👍🏼